modules/core/include/visp3/core/vpArray2D.h

```diff
@@ -54,7 +54,7 @@
   \class vpArray2D
   \ingroup group_core_matrices
 
-  \brief Implementation of a generic 2D array used as vase class of matrices
+  \brief Implementation of a generic column-major 2D array used as vase class of matrices
```

Comment:
```suggestion
  \brief Implementation of a generic 2D array used as base class for matrices
```
Probably `column-major` means here a matrix represented as a 1D array in memory.
`column-major` looks confusing in my opinion, with Fortran vs C++ column-major vs row-major order.